Extensions/xkit_preferences.js

```diff
@@ -1006,7 +1006,7 @@ XKit.extensions.xkit_preferences = new Object({
 
 		if (obj.name.startsWith("xkit_") && XKit.tools.get_setting("xkit_show_internals","false") === "false") { return ""; }
 
-		var blacklisted_extensions = ["xkit_installer"];
+		var blacklisted_extensions = ["xkit_installer","old_stats"];
```

Comment:
is old_sidebar a full replacement to old_stats? I guess it's OK then. But I still think we should tell people since old_sidebar's default looks a lot different than old_stats. I bet that people would start to wonder what happened